internal/git/cmdgit/git.go

```diff
@@ -46,6 +47,15 @@ func (g *Git) run(cmd *exec.Cmd) (string, error) {
 	return stdout.String(), nil
 }
 
+func logGitExecution(cmd *exec.Cmd, stdout *bytes.Buffer, stderr *bytes.Buffer) {
+	logger := log.WithFields(log.Fields{
+		"cmd":    cmd,
+		"stdout": stdout.String(),
+		"stderr": stderr.String(),
+	})
+	logger.Trace("cmdgit")
```

Comment:
nit/style (do as you please with this suggestion).
```suggestion
	log.WithFields(log.Fields{
		"cmd":    cmd,
		"stdout": stdout.String(),
		"stderr": stderr.String(),
	}).Trace("cmdgit")
```